packages/server/modules/workspaces/graph/dataloaders/workspaces.ts

```diff
@@ -35,7 +38,18 @@ const dataLoadersDefinition = defineRequestDataloaders(({ ctx, createLoader }) =
           )
           return ids.map((id) => results[id] || null)
         }
-      )
+      ),
+      getProjectRolesByWorkspaceId: createLoader<
+        { workspaceId: string; userId: string },
+        (Pick<StreamAclRecord, 'role'> & { project: StreamRecord })[] | null
+      >(async (keys) => {
+        const workspaceId = keys[0].workspaceId
```

Comment:
If we're ignoring all the other workspaceIds, then the key is misleading. The loader key makes one think that arbitrary pairs of workspaceId & user Id are all going to be batched as one query. In this case, if you're only supporting 1 workspaceId, but multiple user ids, then you might want to do something similar to `getStreamCommit` - create a separate loader for each workspaceId, so like for getStreamCommit we have `forStream(id)` you might want a `forWorkspace(id)` here